docs/design/chart_format.md

```diff
@@ -394,7 +370,7 @@ The following URI schemes must be supported:
 
 
 #### Special Form Google Storage and S3
-The [Google Storage scheme](https://cloud.google.com/storage/docs/interoperability) (`gs://`)  and the S3 scheme (`s3://`) can also be support. These forms do not follow the URL standard (bucket name is placed in the host field), but their behavior is well-documented.
+The [Google Storage scheme](https://cloud.google.com/storage/docs/interoperability) (`gs://`)  and the S3 scheme (`s3://`) can also be supported. These forms do not follow the URL standard (bucket name is placed in the host field), but their behavior is well-documented.
 
 
 Examples:
```

Comment:
In the user stories below, there's nothing that ties the proposed functionality to features of the product. For example, 

```
As a service dev, I want to be able to work with a team on the same definition
- As a service dev, I want to be able to work with a team on the same definition
- As a service dev, I want to be able to indicate when a particular definition is stable, and how stable it is
- As a service dev, I want to indicate the role I played in building a definition
```

We should not publish user stories without identifying how they are satisfied by the product, except in proposals.